ngsa-asb-dev-westus2.env

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Please remove this file. This is used to store environment variables locally during setup, and should contains secrets we do not want to share publicly.

Reply:
Removed via a commit